provision/ec2_test.go

```diff
@@ -23,3 +23,8 @@ func TestNewEC2ProvisionerFromProfile(t *testing.T) {
 	assert.NotNil(t, prov.client.Config.Credentials)
 	assert.Equal(t, "bob", prov.GetUser())
 }
+
+func TestHomePath(t *testing.T) {
+	homePath := GetHomePath()
+	assert.NotNil(t, homePath)
```

Comment:
`GetHomePath()` returns a string, so the assertion here might be [NotEqual](https://godoc.org/github.com/stretchr/testify/assert#NotEqual) empty string